File: Eulerjl/Euler4.jl
min = 100
max = 999
best = 0
str(x) = "$x"
is_pal(x) = str(x) == reverse(str(x))

for i in min:max
	for j in min:max
		if(is_pal(i*j) && best < i*j)
			best = i*j
		end
	end
end
println(best)	
	

File: Eulerjava/Euler4.java
import java.util.*;

public class Euler4{
	public static void main(String[] args){
		int best = 0;
		for(int i = 100; i < 999; i++){
			for(int j = 100; j < 999; j++){
				if(ispal(i*j) && i*j > best){
					best = i*j;
				}
			}
		}
		System.out.println(best);
	}
	public static boolean ispal(int n){
		String stringy = Integer.toString(n);
		String stringed = new StringBuilder(stringy).reverse().toString();
		return stringy.equals(stringed);
	}
}
